build_opal_bundle.sh

```diff
@@ -2,20 +2,26 @@
 
 set -e
 
-rm -rf custom
-mkdir custom
+# Set default value for PDP_VANILLA
+PDP_VANILLA=${PDP_VANILLA:-"no"}
 
-if [ "$CUSTOM_OPAL" != "" ]
-then
-  echo "Using custom OPAL from $CUSTOM_OPAL"
-	tar -czf custom/custom_opal.tar.gz -C "$CUSTOM_OPAL" --exclude opal-server --exclude '.*' packages README.md
-fi;
+# Check if permit-opa directory already exists
+if [ ! -d "../permit-opa" ]; then
+  # Clone the permit-opa repository into the parent directory if it doesn't exist
+  git clone git@github.com:permitio/permit-opa.git ../permit-opa
+else
+  echo "permit-opa directory already exists. Skipping clone operation."
+fi
 
-if [ "$CUSTOM_OPA" != "" ]
-then
-  echo "Using custom OPA from $CUSTOM_OPA"
+# Conditionally execute the custom OPA tarball creation section based on the value of PDP_VANILLA
+if [ "$PDP_VANILLA" == "no" ]; then
```

Comment:
No is not a good value, better be not empty or bool

Reply:
You're right, thank you